build/Nuget/Sdkproject.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I'm wondering if this could be rolled in to Microsoft.NET.Nuget.proj and we could just invoke Pack target after layout.